### PR TITLE
Make WebComponentsReady fire consistently for Chrome

### LINKED ISSUE #753: WebComponentsReady not firing
In v1.0.0-rc.6 and later, it appears that WebComponentsReady isn't firing on Firefox, using either `webcomponents-lite.js` or `webcomponents-loader.js`. 

In `webcomponents-loader.js`, it doesn't seem to fire at all in Chrome.

JSBin:
https://jsbin.com/yuyuzayewi/edit?html,console,output

### LINKED ISSUE #756: Should fire `WebComponentsReady` on `document`
v0 used to fire it on `document`, v1 fires it on `window` -- let's fire it on `document` and bubble up and please everyone?

### PR DESCRIPTION
**Pull request by azakus**
Also fire WebComponentsReady on document and have it bubble for
backwards compatibility

Fixes #753
Fixes #756

### CODE REVIEW

File: webcomponents-loader.js [hunk 1]
```diff
@@ -45,10 +45,24 @@
...
     // Ensure `WebComponentsReady` is fired also when there are no polyfills loaded.
-    requestAnimationFrame(function() {
-      // Reset the HTMLImports stub.
-      window['HTMLImports'] = null;
-      window.dispatchEvent(new CustomEvent('WebComponentsReady'));
-    });
+    // however, we have to wait for the document to be in 'interactive' state,
+    // otherwise a rAF may fire before scripts in <body>
+
+    var fire = function() {
```
Comment (valdrinkoshi) [inline webcomponents-loader.js]: This logic is required also in https://github.com/webcomponents/webcomponentsjs/blob/fix-wcr/entrypoints/webcomponents-sd-ce-index.js#L21

File: tests/load.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
         window.importsOk = true;
       });
       window.webComponentsReadyCount = 0;
-      window.addEventListener('WebComponentsReady', function() {
+      document.addEventListener('WebComponentsReady', function() {
```
Comment (valdrinkoshi) [inline tests/load.html]: since the event bubbles, do we need to have this listener on document?

File: webcomponents-loader.js (hunk 1 above)
Comment (azakus) [inline webcomponents-loader.js]: 👌

File: tests/load.html (hunk 2 above)
Comment (azakus) [inline tests/load.html]: I'm testing that it does indeed fire on both window and document :)

Comment (valdrinkoshi) [inline tests/load.html]: oh lol, right that makes sense 👍

File: webcomponents-loader.js (hunk 1 above)
Comment (azakus) [inline webcomponents-loader.js]: Done with https://github.com/webcomponents/webcomponentsjs/pull/758/commits/3aa2c3225405e564615ce8b6c2d44e01df2cf048

**Pull request merged**